Automattic/wp-calypso
# Site Settings: Reduxify Monitor card and cleanup

This PR creates a new Monitor card under Security site settings. The old card was not consistent with the new settings design, and was behaving oddly and inconsistently.

The new version looks and functions much alike the rest of the settings cards:
![](https://cldup.com/gkDFC6wAPq.png)

The new card uses Redux, which also allows us to clean up a bunch of legacy code. This PR suggests:

* Updated the card code, ES6ified and altered to use Redux.
* Changed the settings fields to toggles instead of checkboxes.
* Removed the Save Changes button and made the toggles save immediately.
* Replaced the activate/deactivate buttons with a module toggle.
* Updated the layout of the settings to fit the other cards.
* Converted notices to use Redux.
* Deleted and cleaned up:
 * Unused methods from `JetpackSite`
 * Unused `form-base` helper
 * Obsolete `dirty-linked-state` mixin
 * Obsolete site-settings README portion

I'm open to move anything to a separate PR, please let me know if that's necessary.

To test:
* Checkout this branch
* Go to `/settings/security/$site`, where `$site` is a Jetpack site.
* Play with the new monitor card - the main toggle and the settings.
* Verify all works as expected.
* Verify I'm not deleting anything that's still used :laughing:
